internal/pkg/deploy/cloudformation/app.go

```diff
@@ -415,7 +415,8 @@ func (cf CloudFormation) RemoveEnvFromApp(opts *RemoveEnvFromAppOpts) error {
 		if err != nil {
 			return err
 		}
-		if err := cf.s3Client.EmptyBucket(resources.S3Bucket); err != nil {
+		s3 := cf.regionalS3Client(opts.EnvToDelete.Region)
```

Comment:
```suggestion
		s3Client := cf.regionalS3Client(opts.EnvToDelete.Region)
```
in case we conflate with the pkg name